numpy/core/_add_newdocs.py

```diff
@@ -3965,6 +3970,10 @@
     True
     >>> x.tobytes('F')
     b'\\x00\\x00\\x02\\x00\\x01\\x00\\x03\\x00'
+    >>> x.tobytes('A') == x.tobytes()
+    True
+    >>> x.tobytes('K') == x.tobytes()
+    True
```

Comment:
These examples don't really illustrate what the 'K' and 'A' options mean generally, so I'd lean towards not including them.

Reply:
Correcting it.